README.md

```diff
@@ -358,6 +358,11 @@ Convenience method to create a custom font based on a [fontello](http://fontello
 import { createIconSetFromFontello } from 'react-native-vector-icons';
 import fontelloConfig from './config.json';
 const Icon = createIconSetFromFontello(fontelloConfig);
+
+// LineAwesome Example
+import { createIconSetFromFontello } from 'react-native-vector-icons';
+import icoMoonConfig from './config.json';
+const Icon = createIconSetFromFontello(icoMoonConfig, 'LineAwesome', 'line-awesome.ttf');
```

Comment:
Is this necessary as it is already described in your change on line 351?

Reply:
no, not necessary. I understand you wouldn't want bloat the docs but personally I prefer specific examples without any ambiguity. It takes the guesswork out. Happy to remove though

Reply:
I'll leave that up to @oblador, it was just a thought I had 🙂

Reply:
I’m good with either 😊

Reply:
It didn't really make sense to have that example there. I moved the details into the `createIconSetFromFontello` example. [commit](https://github.com/oblador/react-native-vector-icons/pull/818/commits/730e6d127d70d9d744a84358a0279a64a9a67853)